PersonApi.Tests/V1/Gateways/DynamoDbGatewayTests.cs

```diff
@@ -39,6 +42,9 @@ public DynamoDbGatewayTests(AwsIntegrationTests<Startup> dbTestFixture)
             _dynamoDb = dbTestFixture.DynamoDbContext;
             _mockUpdater = new Mock<IEntityUpdater>();
             _logger = new Mock<ILogger<DynamoDbGateway>>();
+            _mockContextAccessor = new Mock<IHttpContextAccessor>();
```

Comment:
is this still needed?

Reply:
No. I thought I'd removed all of that. Will delete.